### ISSUE TITLE
Angular2.beta.17 causes datepicker to fail with a template parse error

### ISSUE THREAD

**Issue by Paladinium** [valor-software/ngx-bootstrap#480]
I'm seeing the following error when using the datepicker:

```
Unhandled Promise rejection:
Template parse errors:
There is no directive with "exportAs" set to "$implicit" ("
  </thead>
  <tbody>
    <template ngFor [ngForOf]="rows" [ERROR ->]#rowz="$implicit" #index="index">
      <tr *ngIf="!(datePicker.onlyCurrentMonth && rowz[0].secondary"): DayPickerComponent@36:37
```

I verified that the datepicker worked with Angular2.beta.16.

**Comment by Paladinium** [comment]
Related to #470, #471, #472, #477

**Comment by valorkin** [comment]
fixed with #481
